Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.py

```diff
@@ -14,6 +15,37 @@ def preprocess_incidents_field(incidents_field):
     return incidents_field
 
 
+def get_phishing_map_labels(comma_values):
+    if comma_values == ALL_LABELS:
+        return comma_values
+    values = [x.strip() for x in comma_values.split(",")]
+    labels_dict = {}
+    for v in values:
+        v = v.strip()
+        if ":" in v:
+            splited = v.split(":")
+            labels_dict[splited[0].strip()] = splited[1].strip()
+        else:
+            labels_dict[v] = v
+    return {k: v for k, v in labels_dict.items()}
+
+
+def build_query_in_reepect_to_phishing_labels(args):
+    mapping = args.get('phishingLabels', ALL_LABELS)
+    query = args.get('query', None)
+    if mapping == ALL_LABELS:
+        return args
+    mapping_dict = get_phishing_map_labels(mapping)
+    tag_field = args['tagField']
+    tags_union = ' '.join(['"{}"'.format(label) for label in mapping_dict])
+    mapping_query = '{}:({})'.format(tag_field, tags_union)
+    if mapping != ALL_LABELS and 'query' not in args:
```

Comment:
mapping will be always != ALL_LABELS because you check it before (you return if it is)